app/web/src/newhotness/layout_components/ComponentSecretAttribute.vue

```diff
@@ -234,6 +243,11 @@ const secretForm = wForm.newForm({
     const propId = props.attributeTree.prop?.id;
     if (!definition) throw new Error("Secret Definition Required");
     if (!propId) throw new Error("Secret Definition Prop Id required");
+    
+    // Stop submission if Name field is missing (no error message shown)
+    if (!value.Name || value.Name.trim() === "") {
+      throw new Error("Name is required");
```

Comment:
This should be in the validator... was there a problem with that?

Reply:
It's less about the visuals--the form shouldn't be allowed to submit without a name. 

The way to do that is by having a validator on the fork to prevent form submission. 

Not allow submission and throw an exception that may prevent other code from running and get reported in observability.

Reply:
I made the change to push this back on the validator and not have the validator show the error